.gitignore

```
@@ -1,2 +1,10 @@
 recast_helloworld/GitHubAction_first_try
 recast_helloworld/Snakemake_action
+reana_helloworld/SnakefileRoot
+reana_helloworld/inputs.yaml
+reana_helloworld/reana-snakemakeRoot.yaml
+reana_helloworld/README.md
+reana_helloworld/report.html
+recast_helloworld/RECAST-instruction.md
+README.md
+README.md
```

```suggestion
README.md
```

Double `README` so can delete this.